var/spack/repos/builtin/packages/py-appnope/package.py

```diff
@@ -0,0 +1,34 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyAppnope(PythonPackage):
+    """Disable App Nap on OS X 10.9"""
+
+    homepage = "https://github.com/minrk/appnope"
+    url      = "https://pypi.io/packages/source/a/appnope/appnope-0.1.0.tar.gz"
+
+    version('0.1.0', '932fbaa73792c9b06754755a774dcac5')
```

Comment:
Okay, we can try (also nobody should try to install it on an non-mac anyways...), but how do I put this when in a ```depends_on``` call? There is no dependency right?
```depends_on(when=sys.platform == 'darwin' and int(platform.mac_ver()[0].split('.')[1]) >= 9)```
seems weird.